dbt_superset_lineage/push_descriptions.py

```diff
@@ -172,11 +173,16 @@ def merge_columns_info(dataset, tables):
     return dataset
 
 
-def put_columns_to_superset(superset, dataset):
+def put_descriptions_to_superset(superset, dataset):
+    logging.info("Putting model descriptions into Superset.")
+
+    payload_description = {'description': dataset['description']}
+    superset.request('PUT', f"/dataset/{dataset['id']}?override_columns=false", json=payload_description)
```

Comment:
Do you think these two `PUT` calls could be merged into one? Something like this:

```python
payload = {'description': dataset['description'], 'columns': dataset['columns_new']}
superset.request('PUT', f"/dataset/{dataset['id']}?override_columns=true", json=payload)
```

Reply:
Of course! It indeed works, let me update it. 🙂